lib/cb/clients/recommendation.rb

```diff
@@ -65,12 +65,15 @@ def self.for_user(*args)
       def self.for_company(company_did)
         my_api = Cb::Utils::Api.instance
         json_hash = my_api.cb_get(Cb.configuration.uri_recommendation_for_company, query: { CompanyDID: company_did })
-
         jobs = []
         if json_hash.key?('Results')
           if json_hash['Results'].key?('JobRecommendation')
-            json_hash['Results']['JobRecommendation']['Jobs'].each do |cur_job|
-              jobs << Models::Job.new(cur_job)
+            if json_hash['Results']['JobRecommendation'].key?('Jobs')
+              if json_hash['Results']['JobRecommendation']['Jobs'].key?('CompanyJob')
+                json_hash['Results']['JobRecommendation']['Jobs']['CompanyJob'].each do |cur_job|
+                  jobs << Models::Job.new(cur_job)
+                end
+              end
             end
```

Comment:
Wouldn't best case of the nested ifs be that you get to skip everything else, thus being faster? With the nested `#fetch`, you have to hit the `#each` function call, even though it just returns.

That said, I think that the `#fetch` approach is prettier and easier to read.